src/containers/Registration.tsx

```diff
@@ -0,0 +1,186 @@
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useState } from "react";
+import { Button, Form, FloatingLabel } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import styled, { css } from "styled-components";
+import * as yup from "yup";
+
+export const windowTitle = "Створіть свій профіль";
+
+const StyledForm = styled(Form)`
+  padding: 0 2rem;
+`;
+
+const StyledFormControl = styled(Form.Control)`
+  margin-top: 1rem;
+  padding: 2rem 0.75rem 2rem 1.7rem !important;
+  height: 5.5rem !important;
+  font-size: 1.5rem;
+
+  ${(props) =>
+    props.top &&
+    css`
+      margin-top: 0rem;
+    `}
```

Comment:
for margin padding you can use bootstrap modificators:
https://getbootstrap.com/docs/4.0/utilities/spacing/

and by using them you don't need to make these conditional styles

Reply:
Fixed